Makefile

```diff
@@ -281,17 +281,20 @@ RALLY_CLIENT_OPTIONS?=basic_auth_user:'admin',basic_auth_password:'changeme'
 RALLY_FLAGS?=--pipeline=benchmark-only --client-options="$(RALLY_CLIENT_OPTIONS)" $(RALLY_EXTRA_FLAGS)
 
 .PHONY: rally
-rally: $(PYTHON_BIN)/esrally rally/corpora/.generated
-	@$(PYTHON_BIN)/esrally race --track-path=rally --kill-running-processes $(RALLY_FLAGS)
+rally: $(PYTHON_BIN)/esrally rally/corpora
+	@$(PYTHON_BIN)/esrally race \
+		--track-path=rally \
+		--track-params=expected_cluster_health:yellow \
```

Comment:
[**For reviewers**] The expected cluster health for local ES cluster will be `yellow` since APM integration requires 2 shards but we only launch 1 ES node in local. I plan to add a separate command to run rally for ESS that will have expected cluster health as `green`.